src/pages/api/newsletter/subscribe.js

```diff
@@ -0,0 +1,23 @@
+export default async function handler(req, res) {
+  if (req.method === 'POST') {
+    await fetch(`https://api.hubapi.com/contacts/v1/contact/?hapikey=${process.env.HUBSPOT_API_KEY}`, {
```

Comment:
You need to validate that the email is also set before triggering the request

- [ ] Check if the email is set
- [ ] Validate that the email is correct (simple regex nothing crazy
- [ ] Return 500 if email is wrong with a nice error message